com.rustdesk.RustDesk.json

```diff
@@ -0,0 +1,110 @@
+{
+  "id": "com.rustdesk.RustDesk",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "rustdesk",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=network",
+    "--filesystem=home",
+    "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/gtk-doc"
+  ],
+  "modules": [
+    "shared-modules/libappindicator/libappindicator-gtk3-12.10.json",
+    {
+      "name": "xdotool",
+      "buildsystem": "simple",
+      "build-commands": [
+        "make -j4 && PREFIX=./build make install",
+        "cp -r ./build/*  /app/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz",
+          "sha256": "96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada"
+        }
+      ]
+    },
+    {
+      "name": "pam",    
+      "buildsystem": "simple",
+      "build-commands": [
+          "./configure --disable-selinux --prefix=/app && make -j4 install"
```

Comment:
THis should use autotools buildsystem, you can drop the manual build commands.

Reply:
done. @bbhtt I hope I have understood you correctly, please check the result. If something is still wrong, please add a code suggestion to the file.